pkgs/applications/networking/gopher/sacc/default.nix

```diff
@@ -23,7 +24,7 @@ stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A terminal gopher client";
-    homepage = "gopher://bitreich.org/1/scm/sacc";
+    homepage = "https://proxy.vulpes.one/gopher/bitreich.org/1/scm/sacc";
```

Comment:
I'm against this: This proxy is operated independently from upstream (as bitreich have chosen not to provide a website via http) and can independently go out of service, change site contents etc.

An URL is an URL, be it http or gopher.

Reply:
@SuperSandro2000 

I'll leave it to you two to decide

Reply:
We are showing the homepage mainly on search.nixos.org and for anyone that wants to quickly visit the link to check something it is no accessible. I am not setting up an ancient protocol just to check a license.

Reply:
I'd prefer floopgap probably since it has been around forever and is more likely to continue to do so, I'd guess.

Still I don't really see the problem, `sacc` is a client for this ancient protocol after all and we track the license in the meta set as well.

Reply:
```suggestion
    homepage = "https://gopher.floodgap.com/gopher/gw?a=gopher%3A%2F%2Fbitreich.org%2F1%2Fscm%2Fsacc";
```

Reply:
Is there somewhere we can get more input?
I feel like this topic is a bit big for GH PRs/Discourse. Is it too small a topic for an RFC?

Reply:
I would ask on discourse.

Reply:
cc-ed you both, I'll drop the homepage change from this version bump until there's been more discussion

Reply:
fine for me. Maybe just revert it to the gopher link.